src/targets/happymodel_2400.ini

```diff
@@ -39,3 +39,12 @@ extends = env:DIY_2400_RX_STM32_CCG_Nano_v0_5_via_STLINK
 
 [env:HappyModel_PP_2400_RX_via_BetaflightPassthrough]
 extends = env:HappyModel_PP_2400_RX_via_STLINK
+
+[env:HappyModel_EP_Dual_2400_RX_via_UART]
```

Comment:
No betaflight passsthrough?

Reply:
Currently not possible :(

PK has some magic ready to add it in 3.1.

Reply:
Gotcha!